
# Topics
  pointCloudTopic: "/middle"               # Point cloud data                       # IMU data
  gpsTopic: "/gps_imu"                   # GPS odometry topic from navsat, see module_navsat.launch file

# Frames
#TODO 1111 remove not used frames
  mapFrame: "map"

  # GPS Settings
  useGpsElevation: true                # if GPS elevation is bad, set to "false"
  gpsCovThreshold: 8                        # m^2, threshold for using GPS data

  # lidar Settings
#  for RS-Ruby-80
  #
  sensor: velodyne                            # lidar sensor type, 'velodyne' or 'ouster' or 'livox' or 'hesai'
  N_SCAN: 80                                  # number of lidar channel (i.e., Velodyne/Ouster: 16, 32, 64, 128, Livox Horizon: 6)
  Horizon_SCAN: 1800                          # lidar horizontal resolution (Velodyne:1800, Ouster:512,1024,2048, Livox Horizon: 4000)

  lidarMinRange: 1.5                          # default: 1.0, minimum lidar range to be used
  lidarMaxRange: 1000.0
  lidarMinRing: 5                          # default: 1.0, minimum lidar range to be used
  lidarMaxRing: 75

  lidarScanDownSample: -1                      #every lidarScanDownSample times cal 1, if -1 drop
  LMOpt_Cnt: 10

  #T_L_B
  extrinsicTrans: [ 0.0, 0.0, -0.5135 ] # -0.5135
  extrinsicRot: [ 0, 1, 0,
                  -1, 0, 0,
                  0, 0, 1 ]
  extrinsicRPY: [ 0, 1, 0,
                  -1, 0, 0,
                  0, 0, 1 ]

#DR 2 lidar
  extrinsicTrans_DR: [ 0.0, 0.0, 0.5135 ] #
  extrinsicRot_DR: [ 0, -1, 0,
                  1, 0, 0,
                  0, 0, 1 ]

  use_drodom_deskew: true
  gtsamGNSSBetweenFactorDistance: 10
  use_gnss_deskew: false
  use_unground_classify: false
